DFHack Dirs.props

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ImportGroup Label="PropertySheets" />
+  <PropertyGroup Label="UserMacros">
+    <dfhack>C:\Users\japam\source\repos\dfhack</dfhack>
```

Comment:
Might want to keep this path out of the repo.

Reply:
It's just a replacement for a similar path that's already in the built bat file. It would have to be changed per-machine in any case.